app/src/main/java/com/codesoom/assignment/TaskHandler.java

```diff
@@ -0,0 +1,137 @@
+package com.codesoom.assignment;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TaskHandler implements HttpHandler {
+    TaskRepository taskRepository = new TaskRepository();
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
이런 것들은 `final`로 선언해줘서 재할당을 금지하는 것을 고려해 보세요.

```suggestion
    final TaskRepository taskRepository = new TaskRepository();
    final ObjectMapper objectMapper = new ObjectMapper();
```

Reply:
c33b6bf86b8dbfaf8e58876b0f9a2607520c93a7 반영했습니다!